src/main/java/eu/europeana/metis/sandbox/common/amqp/RecordMessageConverter.java

```diff
@@ -23,17 +24,19 @@
   private static final String COUNTRY = "country";
   private static final String STATUS = "status";
   private static final String STEP = "step";
+  private static final String EXCEPTION = "exception";
 
   private static final Charset DEFAULT_CHARSET = StandardCharsets.UTF_8;
 
   @Override
   public Message toMessage(Object object, MessageProperties messageProperties)
       throws MessageConversionException {
-    if (!(object instanceof Record)) {
+    if (!(object instanceof Event)) {
       throw new MessageConversionException("Provided object is not of type Record");
     }
 
-    Record record = (Record) object;
+    Event<Record> recordEvent = (Event<Record>) object;
```

Comment:
Perhaps you can change this to 
`Event<?> recordEvent = (Event<?>) object;`
So that we don't get an 'unchecked cast' warning.